### PR TITLE
Copy to devel

### PR DESCRIPTION
**Pull request by bdunne**
:shipit:

### CODE REVIEW

**Comment by Fryguy** [comment]
@simaishi Please review

**Comment by Fryguy** [comment]
@simaishi I'm going to merge so we can get it into tonight's nightly, but please review anyway.

File: scripts/uploader.rb [hunk 1]
```diff
@@ -114,6 +138,11 @@ def uploaded_filename(appliance_name)
       File.basename(filename)
     end
 
+    def devel_filename(appliance_name)
+      name = appliance_name.split("-")
+      (name[0..1] << "devel").join("-") + File.extname(appliance_name)
```
Comment (simaishi) [inline scripts/uploader.rb]: Our GCE image has `.tar.gz` extension `manageiq-gce-master-<date>-<sha>.tar.gz`, so use of `extname` will change to `.gz` and GCE upload won't work.

File: scripts/uploader.rb [hunk 2]
```diff
@@ -52,11 +52,34 @@ def run
...
     end
 
     private
 
+    def master?(filename)
+      filename.include?("-master-")
+    end
+
+    def nightly?
+      !release?
```
Comment (simaishi) [inline scripts/uploader.rb]: We do support `type` other than `nightly` and `release` (although I have never personally used it....), so wouldn't it be better to explicitly check for `type == "nightly"`?

This script is currently called only for nightly/release, so the end result will be same, though...

**Comment by simaishi** [comment]
`-stable` next!

**Pull request merged**